doc/02-Installation.md

```diff
@@ -27,6 +27,7 @@ Requirements
   for MySQL we suggest using at least 5.5.3, for PostgreSQL 9.4.
 * PHP (>= 5.6.3). For best performance please consider use 7.x
 * php-curl
+* php-mbstring (might already be built into your PHP binary)
```

Comment:
It's correct that mbstring is required, but it is intentionally not listed as it is already a requirement for Icinga Web 2.